packages/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "17.0.0",
+  "version": "17.0.1",
```

Comment:
We should not be incrementing core unless we are creating a release after this (which we are not as far as I know).